example/lib/pages/network_tile_provider.dart

```diff
@@ -47,8 +47,10 @@ class NetworkTileProviderPage extends StatelessWidget {
         padding: const EdgeInsets.all(8),
         child: Column(
           children: [
+            // ignore: prefer_const_constructors
             Padding(
               padding: const EdgeInsets.only(top: 8, bottom: 8),
+              // ignore: prefer_const_constructors
```

Comment:
Should add a TODO to remove those ignore in v5

Reply:
https://github.com/fleaflet/flutter_map/pull/1512#discussion_r1191267110